bazel_tools/build_environment.bzl

```
@@ -1,6 +1,15 @@
 # Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+def is_figure(c):
+    return c == "0" or c == "1" or c == "2" or c == "3" or c == "4" or c == "5" or c == "6" or c == "7" or c == "8" or c == "9"
```

I suggest using `in`.

```suggestion
valid_figures = "0123456789"
def is_figure(c):
    return c in valid_figures
```

---

And perhaps "digit" instead of "figure".